src/functions.sh

```diff
@@ -72,17 +72,40 @@ ipt() {
 }
 
 # wrapper to call tc to allow debug logging
-tc_wrapper() {
-    tc_args=$*
-    sqm_trace "${TC_BINARY} $*"
-    ${TC_BINARY} $* >> ${OUTPUT_TARGET} 2>&1
+tc_wrapper(){
+    cmd_wrapper tc ${TC_BINARY} $*
```

Comment:
This should use "$@" instead of $*

See https://stackoverflow.com/questions/2761723/what-is-the-difference-between-and-in-shell-scripts